server/services/workAndSkillsPageService.test.ts

```diff
@@ -90,21 +101,55 @@ describe('WorkAndSkillsService', () => {
           todaysDate,
         )
         expect(res.learnerEducation).toEqual([
-          { key: { text: 'string' }, value: { text: 'Planned end date on 1 March 2023' } },
+          { key: { text: 'GCSE Maths' }, value: { text: 'Planned end date on 31 July 2016' } },
```

Comment:
The local mock data has more realistic data including course names, hence this change